### ISSUE TITLE
#pragma directive not being recognized

### ISSUE THREAD

**Issue by drkameleon** [drkameleon/iswift#15]
Using `#pragma` will throw a parse error.

**Example:**
        
    #pragma mark something

**Comment by drkameleon** [comment]
As of the upcoming 1.3, `#pragma` directives are set to be ignored.

**Comment by bre7** [comment]
Could you add a setting to convert pragma marks to MARKS ?
```objectivec
#pragma mark - Public Getter Methods
```
to
```swift
// MARK: - Public Getter Methods
```

**Comment by drkameleon** [comment]
Hmm... That's definitely an interesting suggestion. Is there some "universally accepted" way of converting `#pragma mark`s? (And the likes...?)

**Comment by bre7** [comment]
Not sure. Pragma marks should be converted to either `// MARK: - xyz` or `// MARK: xyz` depending if you want a separator or not.

**Comment by drkameleon** [comment]
I'll have a look into it. :)

**Comment by bre7** [comment]
ty :D

**Comment by drkameleon** [comment]
> Not sure. Pragma marks should be converted to either // MARK: - xyz or // MARK: xyz depending if you want a separator or not.

Done. (It'll be included in the very next - 1.4 - release). ;-)
